freqtrade/persistence/trade_model.py

```diff
@@ -1232,7 +1232,7 @@ def recalc_trade_from_orders(self, *, is_closing: bool = False):
         current_amount_tr = amount_to_contract_precision(
             float(current_amount), self.amount_precision, self.precision_mode, self.contract_size
         )
-        if current_amount_tr > 0.0:
+        if current_amount_tr >= 0.0:
```

Comment:
I don't think this will work / do what you'd like
This essentially makes the "elif" branch of the conditional non-operational - as `current_amount_tr` is either 0 - or positive.

Reply:
you're right, I'll try to find something else.

Reply:
Well as mentioned in the issue - we'll need a reproduction to test if what you claim is actually true anyway - so i'd start there.
Fixing it in trade-model is however almost certainly the wrong approach.